runtime/doc/motion.txt

```diff
@@ -1083,6 +1083,93 @@ When you split a window, the jumplist will be copied to the new window.
 If you have included the ' item in the 'shada' option the jumplist will be
 stored in the ShaDa file and restored when starting Vim.
 
+When jumpoptions includes "stack", the jumplist behaves like the history in a
```

Comment:
Add a tag here, otherwise the wall-of-text just gets lost in the bigger wall-of-text.

```suggestion
							*jumplist-stack*
When jumpoptions includes "stack", the jumplist behaves like the history in a
```

Is all of the text needed, or can it be explained with less words?

Reply:
The portion describing the default behavior of vim in the browser metaphor was deleted.  It's still a bit lengthy..